dnadoh/analyze.py

```diff
@@ -158,15 +158,33 @@ def update_graph(location):
 
 def parse_args():
     parser = argparse.ArgumentParser()
-    # nothing here for now
     parser.add_argument(
         "--input_stem",
         type=str,
         default=None,
         help="Path/file stem of synthesized data",
         required=True,
     )
+    parser.add_argument(
+        "--isolate_households",
+        type=int,
+        default=True,
+        help="0 or 1 boolean - enforce only one individual per household ID"
+    )
+    parser.add_argument(
+        "--seed", 
+        type=int,
+        default=None,
+        help="RNG seed"
+    )
+    parser.add_argument(
+        "--write-csv",
+        type=int,
+        default=True,
+        help="0 or 1 boolean - write results to csv"
+    )
     options = parser.parse_args()
+    assert options.input_stem is not None, "must specify an input path/file stem"
```

Comment:
technically the `required=True` on L66 should catch an empty argument right (maybe a check we can do is that the input_stem contains a path that actually exists?)

Reply:
Good point. I'm going to just remove it for now, as I think the attempt to load the file path will catch the missing files case.